ci/build-win32.ps1

```diff
@@ -6,11 +6,74 @@ if (-not (Test-Path $subprojects)) {
     New-Item -Path $subprojects -ItemType Directory | Out-Null
 }
 
+# Download pre-built shaderc, it is quite big to build each time
```

Comment:
do we not have build caching?

Reply:
No, not currently. I will try with ccache eventually probably.

For shaderc it is also not that easy to build, so that's another reason.

Reply:
I see.

Reply:
Added ccache, it brings incremental build to 8 minutes, which is not bad.

For shaderc build, like said before I would need to figure it out.